cmd/alerts/add_silence.go

```diff
@@ -0,0 +1,49 @@
+package alerts
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAddSilence() *cobra.Command {
+	return &cobra.Command{
+		Use:   "add-silence <cluster-id> <alertname1> <alertname2>... <comment>",
+		Short: "add a new silence",
+		Long:  `add new silence`,
```

Comment:
For the long description, I'd like to see some more details about the command. What does it mean to "add a new silence"? How long does it last for? Etc.